deepfence_ui/app/scripts/components/vulnerability-view/registry-scan/credentials-form.js

```diff
@@ -132,16 +139,23 @@ class ContainerImageRegistryCredentialsForm extends React.PureComponent {
               return null;
             }
             return (
-              <Field
-                key={field.name}
-                name={field.name}
-                component={fieldTypeRenderer(field.type)}
-                fileName={fileValues[field.name] ? fileValues[field.name].name : ''}
-                labelBefore={field.label}
-                filenamePosition="before"
-                validate={validateJsonFileType}
-                {...field}
-              />
+              <>
+                {
+                  field.name === 'non_secret.target_account_role_arn' && enableField ? 
```

Comment:
do we also need to check if the form is for ecr?

Reply:
added